migrations/2024-04-10-034442_alter_payout_status/down.sql

```diff
@@ -0,0 +1,5 @@
+DELETE FROM pg_enum
```

Comment:
It's not encouraged to delete a value from pg_enum (we can just use `select 1` here), anyways we are using `if not exists` while adding the value